### PR TITLE
Ice berg feature

### PR DESCRIPTION
**Pull request by ice949**
### New Features ###

- Re-created your Awesome books app with ES-6 syntax
- Displayed the current date below the navigation bar using luxon module

### CODE REVIEW

File: modules/app.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  const ui = new CreateUI();
+
+  function getVal(title, author, id) {
+    return new AwesomeBooks(title, author, id);
+  }
+
+  // important to hold length of local  storage
+  let id = 0;
+
+  function getIndexInLocalStorage() {
```
Comment (omar-labana) [inline modules/app.js]: - [ ] Kindly make sure to use arrow syntax for functions

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @ice949, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/app.js (hunk 1 above)
Comment (ice949) [reply modules/app.js]: @omar-labana Noted with thanks

**Review by Mre55 — APPROVED** [review]
**Hi @ice949,**

# Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/app.js (hunk 1 above)
Comment (ice949) [reply modules/app.js]: @Mre55 thanks for review

**Pull request merged**